lp-steam/templates/csgo/server.j2

```diff
@@ -22,14 +22,34 @@ sv_lan 1
 host_players_show 2
 host_info_show 2
 
+game_type 0 
+game_mode 1
+
 {% if item.value.gotv | default(false)  %}
+tv_port "{{ item.value.gameport + 5 }}" 
 tv_enable "1"
 tv_title "{{ lp.name }} GOTV {{ steam_server_id }}.{{ item.key }}"
 tv_name "{{ lp.name }} GOTV {{ steam_server_id }}.{{ item.key }}"
 tv_password "{{ csgo_gotv_password | default('stream') }}"
 tv_maxclients "{{ csgo_gotv_maxclients | default(4) }}"
 tv_snapshotrate "{{ csgo_gotv_snaprate | default(64) }}"
-tv_delaymapchange 1
+tv_delaymapchange "1" 
+tv_autorecord "1" 
+tv_allow_camera_man "1" 
```

Comment:
This var doesn't exist, but `tv_allow_camera_man_steamid` does, please check if we need this.